app/src/main/java/com/example/instasnap/Login.java

```diff
@@ -0,0 +1,117 @@
+package com.example.instasnap;
+
+import androidx.annotation.NonNull;
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.text.TextUtils;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.ProgressBar;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import com.google.android.gms.tasks.OnCompleteListener;
+import com.google.android.gms.tasks.Task;
+import com.google.firebase.auth.AuthResult;
+import com.google.firebase.auth.FirebaseAuth;
+import com.google.firebase.auth.FirebaseUser;
+
+public class Login extends AppCompatActivity {
+
+    private EditText editTextEmail;
```

Comment:
fix to: private EditText _editTextEmail
-- all of the private variables

Reply:
Done